specification/automation/resource-manager/Microsoft.Automation/preview/2024-08-01-preview/locations.json

```diff
@@ -0,0 +1,163 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "AutomationManagement",
+    "version": "2024-08-01-preview",
+    "x-ms-code-generation-settings": {
+      "useDateTimeOffset": true
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Automation/locations/{locationName}/usages": {
+      "get": {
+        "tags": [
+          "Usages"
+        ],
+        "operationId": "Usages_ListByLocation",
+        "description": "Retrieve the usage for the location.",
+        "externalDocs": {
+          "url": "http://aka.ms/azureautomationsdk/getUsagesOfLocation"
+        },
+        "x-ms-examples": {
+          "List usages": {
+            "$ref": "./examples/getUsagesOfLocation.json"
+          }
+        },
+        "parameters": [
+          {
+            "name": "locationName",
```

Comment:
Can we reference LocationParameter from common-types ?
https://github.com/Azure/azure-rest-api-specs/blob/a651ba25cda4eec698a3a4e35f867ecc2681d126/specification/common-types/resource-management/v6/types.json#L655

Reply:
Makes sense done the changes